src/3box.js

```diff
@@ -173,10 +173,18 @@ class ThreeBox {
     }
   }
 
+  /**
+   * Uninitializes the 3box. Should be called after you are done with
```

Comment:
Uninitializes the user's 3Box without clearing the local cache. Should be called after you are done using the 3Box instance, but without logging the user out.